cli.go

```diff
@@ -421,4 +426,7 @@ Options:
 
 -debug
 	Enable debug output
+
+-generatenotes
+	Generate the body of the release automatically based on .github/release.yml
```

Comment:
I didn't known it is configurable!

https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes

Why not add the link to the document?

Reply:
> Why not add the link to the document?

Added mention with link in README.md